pkgs/applications/video/mkvtoolnix/default.nix

```diff
@@ -131,7 +136,6 @@ stdenv.mkDerivation rec {
     homepage = "https://mkvtoolnix.download/";
     license = licenses.gpl2Only;
     maintainers = with maintainers; [ codyopel rnhmjoj ];
-    platforms = platforms.linux
-      ++ optionals (!withGUI) platforms.darwin;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
I was wondering about that because I've seen packages using both forms and I don't know which one to use.

I will update the derivation.

For the record:

```
$ rg 'platforms\.linux \+\+ platforms\.darwin' | wc -l
     213
$ rg 'platforms\.darwin \+\+ platforms\.linux' | wc -l
       9
```